client/state/audio/middleware.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_RECEIVE_EVENT,
+} from 'state/action-types';
+
+export const playSound = src => {
+	if ( typeof Audio !== 'function' ) {
+		// No Audio support in this browser
+		return;
+	}
```

Comment:
Cool, thanks for the heads up. I'll add the conditional injection once #10897 is merged.